assets/performanceprofile/tuned/openshift-node-performance-intel-x86

```diff
@@ -0,0 +1,25 @@
+[main]
+summary=Platform specific tuning for Intel x86
+
+[bootloader]
+cmdline_iommu_intel=intel_iommu=on iommu=pt
+
+{{if .PerPodPowerManagement}}
+cmdline_pstate=intel_pstate=active
+{{else if .HardwareTuning}}
+cmdline_pstate=intel_pstate=passive
+{{else}}
```

Comment:
I think these are backwards

Reply:
I'm not sure how I managed that but it does appear I inverted them from the original at some point, will fix